UPGRADING.md

```diff
@@ -1,5 +1,21 @@
 # Upgrade guidelines
 
+## 0.43.1 to 0.43.2
+
+[#375 Resource detection processor is configured to override all host and cloud
+attributes](https://github.com/signalfx/splunk-otel-collector-chart/pull/375)
+
+If you still want to keep the previous behavior, use the following custom
+values.yaml configuration:
+
+```yaml
+agent:
+  config:
+    processors:
+      resourcedetection:
+        override: false
```

Comment:
Does the resource processor's new `upsert` prevent complete rollback?

Reply:
`--set 'agent.config.processors.resource.attributes[0].action=insert'` doesn't seem to properly merge:

```
      resource:
        attributes:
        - action: insert
```

Reply:
Not sure I understand what do you mean by complete rollback. `upsert` in resource processor is needed to keep existing behavior of setting `k8s.cluster.name` attribute -- it should be set from `clusterName` values.yaml config option, and `k8s.cluster.name` set by resourcedetector should be ignored.

Reply:
`k8s.cluster.name` is being set by resourcedetection processor only in some versions of GKE. Making sure this is taken care of in this rollback example, we would need to add something like:

```yaml
  config:
    processors:
      resourcedetection:
        override: false
      resource:
        attributes:
          - action: insert
            key: k8s.node.name
            value: "${K8S_NODE_NAME}"
          - action: upsert
            key: k8s.cluster.name
            value: < your cluster name from .Values.clusterName >
          < extra attribute from .Values.extraAttributes.custom >
```

I'm not sure likelihood of changed `k8s.cluster.name` warrants adding this config. We can probably just mention that `k8s.cluster.name` can be changed in GKE

Reply:
Hmm... Actually `upsert` is good. It doesn't need to be changed for the rollback. It preserves the required `k8s.cluster.name` value set from `clusterName` before and after the change.